src/base/preferences.h

```diff
@@ -252,7 +252,13 @@ class Preferences : public QObject
     void setAutoRunEnabled(bool enabled);
     QString getAutoRunProgram() const;
     void setAutoRunProgram(const QString &program);
+    bool isStartAutoRunEnabled() const;
+    void setStartAutoRunEnabled(bool enabled);
+    QString getStartAutoRunProgram() const;
+    void setStartAutoRunProgram(const QString &program);
 #if defined(Q_OS_WIN)
+    bool isStartAutoRunConsoleEnabled() const;
```

Comment:
Does it really make sense to enable/disable console for each "auto run" case independently?

Reply:
I have not the faintest idea, for I do not use Windows. :)
I'd suggest this be asked from those needing the console (for non-debugging/checking purposes) at all.

Reply:
Just let me know if I would be to remove one (if possible, with recommendation on where to move the remaining option on the UI, please, as surely it wouldn't be logical to remain wherever it is now)